Content.Server/Hands/Systems/HandsSystem.cs

```diff
@@ -223,7 +225,7 @@ private bool HandleThrowItem(ICommonSession? session, EntityCoordinates coords,
                 playerEnt.IsInContainer() ||
                 !playerEnt.TryGetComponent(out SharedHandsComponent? hands) ||
                 !hands.TryGetActiveHeldEntity(out var throwEnt) ||
-                !_interactionSystem.TryThrowInteraction(hands.Owner, throwEnt))
+                !_actionBlockerSystem.CanThrow(playerEnt.Uid))
                 return false;
```

Comment:
Somewhat unrelated, but previously this function would trigger the throw interactions twice, leading to two throwing logs per throw.